entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
#!/usr/bin/env bash

Reply:
> @bfabio THANK YOU!
> 
> I've added some minor comments. Also, few questions:
> 
>     * What's dot env for (in this context)? I'm asking because the image should be one. Vars should then get values at run-time (at client-side). What's the point of reading vars for a .env file at build time? Is it for development?

Exactly, plus `build` and `build-prod`. I figured that we presumably don't want to break those and still provide the users a way to build and use the app without necessarly depend on Docker.

>     * I see some updates in the yarn.lock (besides the ones related to dovenv). Should we address them separately?

Nice catch. That's related to #132 and e7b5cff847ea9f1b47014 where npm was run instead of yarn. I'll open a PR with `yarn.lock` changes for e7b5cff847ea9f1b47014. :+1: 

>     * Can you please confirm you've tried the patch?

I did. The docker image builds and runs as expected and I played around with the app (on a fairly superficial level though).

Reply:
> #!/usr/bin/env bash

I'm getting over that habit (and I have to force myself to not use `env` :sweat_smile:) after https://www.debian.org/doc/packaging-manuals/python-policy/ch-python.html and especially so for Docker images where we *know* where `bash` is.

Is there something I missing?

Reply:
Nope, I didn't know that. Thanks for the pointer. I'll go deeper through it ;)